.github/workflows/positron-python-nightly.yml

```
@@ -70,6 +73,13 @@ jobs:
       - name: Run Python unit tests
         run: python python_files/tests/run_all.py
```

this can be updated too, something like this

```suggestion
        run: python python_files/tests/run_all.py --junit-xml= python-unit-test-results.xml
```